js/services/JobDetailsService.js

```diff
@@ -27,6 +27,24 @@ define(['appConfig', 'services/job/jobDetail', 'atlas-state', 'services/http'],
 			}
 		}
 	}
+	
+	function setJobListMutated() {
+		sharedState.jobListing.valueHasMutated();
+	}
+
+	function addPoll(opts) {
```

Comment:
I still see direct usages of the `PollService.add`:
![image](https://user-images.githubusercontent.com/10010071/71668520-40b0c800-2d37-11ea-912d-89e7c9d9573b.png)

However, the general issue that concerns me is the following: it becomes unclear which service to use. Either the `PollService.add` should become "private" so that developers do not attempt to use it directly, OR, in my opinion, the preferrable solution is to throw event from the `PollService` and process the event in the JobDetails. So that we have loose coupling, avoid ambigioty in services and keep services atomic

Reply:
The following points bother me 
1) PollService now has a new responsibility `isJobListMutated`. This contradicts his original idea, just executes callback every N seconds. These processes must be independent 
2) user-bar.js also uses a pollService which for some reason does not work. I would like to understand what is wrong with it or:
3) As an alternative for isJobListMutated it is possible to use events, as I know Vlad has added eventemitter3 framework( I guess we are not merge it yet), Events make the code clearer.